test/core/GnosisSafe.FallbackManager.spec.ts

```diff
@@ -70,6 +70,19 @@ describe("FallbackManager", async () => {
 
         })
 
+        it('emits event when is set', async () => {
+            const { safe } = await setupWithTemplate()
+            const handler = await defaultCallbackHandlerDeployment()
+
+            // Setup Safe
+            await safe.setup([user1.address, user2.address], 1, AddressZero, "0x", AddressZero, AddressZero, 0, AddressZero)
+
+            // Check event
+            await expect(
+                executeContractCallWithSigners(safe, safe, "setFallbackHandler", [handler.address], [user1])
+            ).to.emit(safe, "ChangedFallbackHandler").withArgs(handler.address)
```

Comment:
Couldn't we just add this check to the one above? instead of having a completely separate test?

Reply:
Of course, I just didn't want to mess with your tests :)